GitHub Codespaces Session Fails to Build

### Prerequisites

- [X] I'm using the latest version of `near-api-js`.
- [X] I have tried to start with a fresh project and reproduce the defect with minimal code changes.
- [X] I have read the console error messages carefully (if applicable).

### Description

When running the repository from a new codespaces VM, `pnpm` fails to build after installing dependencies. 

Specifically, it produces this output:

```bash
@idea404 ➜ /workspaces/near-api-js (master) $ pnpm i && pnpm build
Scope: all 15 workspace projects
​
   ╭─────────────────────────────────────────────────────────────────╮
   │                                                                 │
   │                Update available! 8.1.0 → 8.3.1.                 │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v8.3.1   │
   │                Run "pnpm add -g pnpm" to update.                │
   │                                                                 │
   │     Follow @pnpmjs for updates: https://twitter.com/pnpmjs      │
   │                                                                 │
   ╰─────────────────────────────────────────────────────────────────╯
​
 WARN  Moving @changesets/changelog-github that was installed by a different package manager to "node_modules/.ignored"
 WARN  Moving @changesets/cli that was installed by a different package manager to "node_modules/.ignored"
 WARN  Moving @commitlint/cli that was installed by a different package manager to "node_modules/.ignored"
 WARN  Moving @commitlint/config-conventional that was installed by a different package manager to "node_modules/.ignored"
 WARN  Moving @typescript-eslint/eslint-plugin that was installed by a different package manager to "node_modules/.ignored"
 WARN  7 other warnings
Downloading registry.npmjs.org/typescript/4.9.4: 11.6 MB/11.6 MB, done
packages/near-api-js                     |  WARN  deprecated querystring@0.2.0
packages/accounts                        |  WARN  deprecated sane@4.1.0
packages/accounts                        |  WARN  deprecated source-map-resolve@0.5.3
packages/accounts                        |  WARN  deprecated resolve-url@0.2.1
packages/accounts                        |  WARN  deprecated source-map-url@0.4.1
packages/accounts                        |  WARN  deprecated urix@0.1.0
packages/accounts                        |  WARN  deprecated w3c-hr-time@1.0.2
Downloading registry.npmjs.org/turbo-linux-64/1.6.3: 6.16 MB/6.16 MB, done
 WARN  GET https://registry.npmjs.org/reusify/-/reusify-1.0.4.tgz error (ERR_PNPM_FETCH_522). Will retry in 10 seconds. 2 retries left.
Packages: +1043
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: /workspaces/.pnpm-store/v3
  Virtual store is at:             node_modules/.pnpm
Progress: resolved 1054, reused 0, downloaded 1043, added 1043, done
node_modules/.pnpm/core-js@3.26.1/node_modules/core-js: Running postinstall script, done in 166ms
node_modules/.pnpm/turbo@1.6.3/node_modules/turbo: Running postinstall script, done in 145ms
node_modules/.pnpm/cbor-extract@2.1.1/node_modules/cbor-extract: Running install script, done in 222ms
. prepare$ husky install
│ husky - Git hooks installed
└─ Done in 110ms
​
devDependencies:
+ @changesets/changelog-github 0.4.7
+ @changesets/cli 2.25.2
+ @commitlint/cli 17.3.0
+ @commitlint/config-conventional 17.3.0
+ @typescript-eslint/eslint-plugin 5.46.1
+ @typescript-eslint/parser 5.46.1
+ commitlint 17.3.0
+ eslint 8.29.0
+ husky 7.0.4
+ rimraf 3.0.2
+ turbo 1.6.3
+ typescript 4.9.4 (5.0.4 is available)
​
Done in 1m 2.4s
​
> @near-js/monorepo@ build /workspaces/near-api-js
> turbo run build
​
 ERROR  run failed: workspace configuration error: package.json: no workspaces found. Turborepo requires npm workspaces to be defined in the root package.json
 ELIFECYCLE  Command failed with exit code 1.
 ```

### Reproducible demo

_No response_

### Steps to reproduce

1. Create a codespaces VM on `master`
2. Run `pnpm i && pnpm build`

### Expected behavior

Dependencies should be installed and repo should build.

### Actual behavior

Exit with code 1.

### Your environment

Standard codespaces OS and env.

### Self-service

- [X] I'd be willing to fix this bug myself.

--- a/.devcontainer/devcontainer.json
+++ b/.devcontainer/devcontainer.json
@@ -1,0 +1,6 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
+  "features": {
+    "ghcr.io/devcontainers/features/node:1": {}
+  }
+}

